sim/scenarios/droplist/README.md

```diff
@@ -0,0 +1,38 @@
+# Droplist
+
+This scenario uses a bottleneck link similar to the [../simple-p2p](simple-p2p)
+scenario and optionally allows configuring the link to drop specific packets in
+either direction.
+
+This scenario can be used to test the performance of QUIC implementations under
+specific loss patterns. For instance, tests can:
+* drop specific handshake packets,
+* drop tail packets,
+* drop important control packets, such as path validation packets.
```

Comment:
How would you know what the packet number of the path validation packet is?

Reply:
Same as above. The point here is that you can drop specific packets. You'd have to find those packet numbers somehow, and there's some trial and error involved.